Improve `dbms.databases.seed_from_uri_providers` description (#1954)
neo4j/docs-operations: modules/ROOT/pages/configuration/configuration-settings.adoc

|Description

> a|Databases may be created from an existing _seed_ (a database backup or dump) stored at some source URI.

Reviewer: ```suggestion
a|Databases may be created from an existing _seed_ (a database backup or dump) stored at a specific source URI.
```

> Different types of seed source are supported by different implementations of `com.neo4j.dbms.seeding.SeedProvider`.

Reviewer: ```suggestion
Different implementations of `com.neo4j.dbms.seeding.SeedProvider` support various types of seed sources.
```

> The following values are available, `CloudSeedProvider`, `S3SeedProvider`, `FileSeedProvider` and `URLConnectionSeedProvider`.

Reviewer: ```suggestion

The following values are available: `S3SeedProvider`, `CloudSeedProvider`, `URLConnectionSeedProvider`, and `FileSeedProvider`.
```

> Seeds addressed with  `s3` are supported by `S3SeedProvider`.

Reviewer: ```suggestion

* `S3SeedProvider` supports seeds addressed with  `s3`.
```

> Seeds addressed with  `s3`, `azb`, `gs` are supported by `CloudSeedProvider`.

Reviewer: ```suggestion
* `CloudSeedProvider` supports seeds addressed with  `s3`, `azb`, `gs`.
```

> Seeds addressed with `ftp`,`http`, and `https` URIs are supported by `URLConnectionSeedProvider`.

Reviewer: ```suggestion
* `URLConnectionSeedProvider` supports seeds addressed with `ftp`,`http`, and `https`.
```

> Seeds addressed with `file` are supported by `FileSeedProvider`.

Reviewer: ```suggestion
* `FileSeedProvider` supports seeds addressed with `file`.

```
